### PR TITLE
les 95 Create Login page

### PR DESCRIPTION
**Pull request by Kurilo-Pavel**

### CODE REVIEW

File: src/components/Email.tsx [hunk 1]
```diff
@@ -1,39 +1,52 @@
...
+      <input type="text"
+             name="email"
+             id="email"
+             required
+             minLength={3}
+             maxLength={256}
+             className={"Email-Input" + (error ? " invalid-Email-Input" : "")}
+             placeholder="username@gmail.com"
+             value={value}
+             onKeyUp={handleChanges}{...field}
```
Comment (Terapool) [inline src/components/Email.tsx:44]: Please, move {...field} to a new line

File: src/components/modal/modal.scss [hunk 2]
```diff
@@ -1,21 +1,29 @@
+body{
```
Comment (Terapool) [inline src/components/modal/modal.scss:1]: Please enter space before {

body{ -> body {

check all cases

File: src/pages/LoginPage.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    .max(256, 'Password should be less than 256 characters'),
+  termsAndConditions: Yup.boolean().isTrue(
+    'You should accept our terms and conditions'
+  ),
+})
+
+const LoginPage = () => {
+  const [isVisible, setIsVisible] = useState(true);
+
+  const showModal = () => {
```
Comment (Terapool) [inline src/pages/LoginPage.tsx:30]: rename showModal to 'toggleModalVisibility'

File: src/components/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import "./button.scss"
+
+type ButtonProps = {
+  buttonType: 'button' | 'submit' | 'reset' | undefined;
+  buttonText: string;
+}
```
Comment (Terapool) [inline src/components/Button.tsx:6]: You can add onClick property

**Review by valer-sky — APPROVED** [review]

**Review by Terapool — APPROVED** [review]

**Review by NatashaMikhaltsova — APPROVED** [review]

**Review by OxygenO22 — APPROVED** [review]

**Pull request merged**